misc/handlers.go

```diff
@@ -140,17 +141,55 @@ func handlerAbout(w http.ResponseWriter, rq *http.Request) {
 var stylesheets = []string{"default.css", "custom.css"}
 
 func handlerStyle(w http.ResponseWriter, rq *http.Request) {
-	w.Header().Set("Content-Type", mime.TypeByExtension(".css"))
+	type fileData struct {
+		Content io.Reader
+		ModTime time.Time
+	}
+	var filesData []fileData
+
+	var latestModTime time.Time
+
+	// Step 1: Collect files data and determine the latest modification time
 	for _, name := range stylesheets {
 		file, err := static.FS.Open(name)
 		if err != nil {
 			continue
 		}
-		_, err = io.Copy(w, file)
+
+		fileStats, err := file.Stat()
 		if err != nil {
+			continue
+		}
+
+		modTime := fileStats.ModTime()
```

Comment:
How it works for embedded files?

static.FS.Open(name) returns a file either from embedded or from the file system

I'm not sure that this point will be worked out correctly here.